site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -87,6 +87,17 @@ Configurable parameters:
 - agones_version - the version of agones to install (default is the latest version from the [Helm repository](https://agones.dev/chart/stable))
 - machine_type - machine type for hosting game servers (default is "n1-standard-4")
 - node_count - count of game server nodes for the default node pool (default is "4")
+- network - the name of the network you want your cluster and firewall rules to reside (default is "default")
+
+{{% alert title="Warning" color="warning"%}}
+On the lines that read `source = "git::https://github.com/googleforgames/agones.git//install/terraform/modules/gke/?ref=master"`
```

Comment:
I think @aLekSer had left this pointed at the master branch (rather than a release branch) on purpose.

Reply:
Working on that, will provide more details soon.

Reply:
This seems like a brittle approach to me? How do we ensure that we don't accidentally break production systems with development changes?

Reply:
Well we can use both options. The master branch was used so that anyone can play and test `terraform apply` with those modules before the release.

Reply:
Seems like we keep optimising for our development, rather than our users -- we should really be optimising for our users.

We can build extra tools for ourselves to handle our development scenarios, but we shouldn't be foisting that onto our users, in my humble opinion.